apps/web/src/pages/index.tsx

```diff
@@ -1,9 +1,13 @@
 import type { NextPage } from 'next';
 import 'reflect-metadata';
-import { Sidebar } from '../components/Sidebar';
+import { CareLocationSelector } from '../components/CareLocationSelector';
 
 const Home: NextPage = () => {
-  return <Sidebar></Sidebar>;
+  return (
+    <div>
+      <CareLocationSelector></CareLocationSelector>
```

Comment:
We should wait for @PeterPavlatos's PR to be merged before merging this

Reply:
Sure thing, would make sense.